tests/test_utils.py

```diff
@@ -147,10 +145,58 @@ def test_file_path_validator_with_invalid_paths(actual_path):
 def test_file_path_validator_with_valid_paths(actual_path):
     from prompt_toolkit.document import Document
 
-    validator = rasa.utils.io.questionary_file_path_validator(
-        [".yml", ".yaml"], "error message"
-    )()
+    validator = rasa.utils.io.file_type_validator([".yml", ".yaml"], "error message")
 
     document = Document(actual_path)
     # If the path is valid there shouldn't be an exception
     assert validator.validate(document) is None
+
+
+@pytest.mark.parametrize("input", ["", "   ", "\t", "\n"])
+def test_non_empty_text_validator_with_empty_input(input):
```

Comment:
`input` shadows a python function. Can we rename it? e.g. `to_validate` or `user_input`

Reply:
yep, will use `user_input`